app/models/user_category.rb

```diff
@@ -25,6 +25,13 @@ class UserCategory < ApplicationRecord
   has_many :sub_categories, through: :expense_categories
   has_many :transactions, dependent: :destroy
 
+  def is_Valid?
+    name!="Personal"
+  end
+  
+  def valid_user?
```

Comment:
```suggestion
  def user_nil?
```

method name should be follow the method working or scenarios